pkg/runtime/runtime.go

```diff
@@ -414,7 +536,7 @@ func (a *DaprRuntime) initRuntime(ctx context.Context) error {
 
 	err = a.startGRPCAPIServer(a.daprGRPCAPI, a.runtimeConfig.apiGRPCPort)
 	if err != nil {
-		log.Fatalf("failed to start API gRPC server: %s", err)
+		return fmt.Errorf("failed to start API gRPC server: %s", err)
```

Comment:
```suggestion
		return fmt.Errorf("failed to start API gRPC server: %w", err)
```

Reply:
All of these changes also allow detecting if the error is context.Canceled for example